manifests/database.pp

```diff
@@ -0,0 +1,31 @@
+# Set up the pulp database
+class pulp::database {
+  if $pulp::db_manage {
+    if (versioncmp($::mongodb_version, '2.6.5') >= 0) {
+      $mongodb_pidfilepath = '/var/run/mongodb/mongod.pid'
+    } else {
+      $mongodb_pidfilepath = '/var/run/mongodb/mongodb.pid'
+    }
+
+    class { '::mongodb::server':
+      pidfilepath => $mongodb_pidfilepath,
+    }
+
+    Service['mongodb'] -> Service['pulp_celerybeat']
+    Service['mongodb'] -> Service['pulp_workers']
+    Service['mongodb'] -> Service['pulp_resource_manager']
+    Service['mongodb'] -> Exec['migrate_pulp_db']
+  }
+
+  exec { 'migrate_pulp_db':
+    command     => 'pulp-manage-db',
+    path        => '/bin:/usr/bin',
+    logoutput   => 'on_failure',
+    user        => 'apache',
+    refreshonly => true,
+    require     => File['/etc/pulp/server.conf'],
+  }
+
+  Class['pulp::install'] ~> Exec['migrate_pulp_db'] ~> Class['pulp::service']
```

Comment:
I don't understand what you mean.

if it's about Class['pulp::install'] and lass['pulp::config'], I wanted to force a migrate_pulp_db if any package is installed/updated or any configuration changes.